app/views/browse/deployment.html

```diff
@@ -108,8 +110,11 @@ <h1 class="contains-actions">
                           <th>Created</th>
                         </tr>
                       </thead>
-                      <tbody>
-                        <tr ng-repeat="replicaSet in replicaSetsForDeployment">
+                      <tbody ng-if="(replicaSetsForDeployment | hashSize) == 0">
```

Comment:
`replicaSetsForDeployment` is an array right? `hashSize` won't work on arrays. The `size` filter we have does, and we're trying to switch code to use it for that reason.

```html
<tbody ng-if="!(replicaSetsForDeployment | size)">
```